examples/v1beta1/README.md

```diff
@@ -1,391 +1,141 @@
+
+- [Tree of Parzen Estimators (TPE)](./hp-tuning/tpe.yaml)
+
+- [Multivariate TPE](./hp-tuning/multivariate-tpe.yaml)
+
+- [Covariance Matrix Adaptation Evaluation Strategy (CMA-ES)](./hp-tuning/cma-es.yaml)
+
+- [Sobol's Quasirandom Sequence](./hp-tuning/sobol.yaml)
+
+- [HyperBand](./hp-tuning/hyperband.yaml)
+
+### Neural Architecture Search
+
+Check the [Neural Architecture Search](https://www.kubeflow.org/docs/components/katib/overview/#neural-architecture-search)
+Experiments for the following algorithms:
+
+- [Efficient Neural Architecture Search (ENAS)](./nas/enas-gpu.yaml)
+
+- [Differentiable Architecture Search (DARTS)](./nas/darts-gpu.yaml)
+
+### Early Stopping
+
+Improve your Hyperparameter Tuning Experiments with the following
+[Early Stopping](https://www.kubeflow.org/docs/components/katib/early-stopping/) algorithms:
+
+- [Median Stopping Rule](./early-stopping/median-stop.yaml)
+
+## Katib Python SDK Examples
+
+To learn more about Katib Python SDK check [this directory](./sdk).
+
+## Resume Katib Experiments
+
+You can use different resume policies in Katib Experiments. Follow
+[this guide](https://www.kubeflow.org/docs/components/katib/resume-experiment/)
+to know more about it. Check the following examples:
+
+- [Resume From Volume](./resume-experiment/from-volume-resume.yaml)
+
+- [Never Resume](./resume-experiment/never-resume.yaml)
+
+## Metrics Collector
+
+Katib supports the various metrics collectors and metrics strategies.
+Check the [official guide](https://www.kubeflow.org/docs/components/katib/experiment/#configuration-spec)
+to know more about it. In this directory you can find the following examples:
+
+- [File Metrics Collector](./metrics-collector/file-metrics-collector.yaml)
+
+- [Custom Metrics Collector](./metrics-collector/custom-metrics-collector.yaml)
+
+- [Metrics Collection Strategy](./metrics-collector/metrics-collection-strategy.yaml)
+
+## TODO (andreyvelich) Discuss about this name. Trial Settings
+
+You can specify different settings for your Trial template. To know more about it
+follow [this guide](https://www.kubeflow.org/docs/components/katib/trial-template/#use-trial-template-to-submit-experiment).
+Check the following examples:
+
+- [Trial with ConfigMap Source](./trial-settings/trial-configmap-source.yaml)
+
+- [Trial with Metadata Substitution](./trial-settings/trial-metadata-substitution.yaml)
+
+## TODO (andreyvelich) Discuss about this name. Trial Training Containers
+
+Check the following examples for Trial training containers:
+
+- [MXNet MNIST](./trial-training-containers/mxnet-mnist)
+
+- [PyTorch MNIST](./trial-training-containers/pytorch-mnist)
+
+## Katib with Kubeflow Training Operator
+
+Katib has out of the box support for the [Kubeflow Training Operators](https://github.com/kubeflow/tf-operator) to
+perform Trial's [Worker job](https://www.kubeflow.org/docs/components/katib/overview/#trial).
+Check the following examples for the various distributed operators:
```

Comment:
I think we also supported the previous versions for some of them.
For example, TFJob and PyTorchJob was integrated in 2018 in Katib.

Reply:
It makes sense.
I think It needs to describe the training-operator version since I guess it only works with train-operator >= 1.3 in the following examples. WDYT @andreyvelich ?

https://github.com/kubeflow/katib/blob/9f15ec5774f8d590e085275a0f541e6f03c943e9/examples/v1beta1/kubeflow-training-operator/mpijob-horovod.yaml#L41-L43

https://github.com/kubeflow/katib/blob/9f15ec5774f8d590e085275a0f541e6f03c943e9/examples/v1beta1/kubeflow-training-operator/pytorchjob-mnist.yaml#L36-L38

https://github.com/kubeflow/katib/blob/9f15ec5774f8d590e085275a0f541e6f03c943e9/examples/v1beta1/kubeflow-training-operator/tfjob-mnist-with-summaries.yaml#L43-L45

https://github.com/kubeflow/katib/blob/9f15ec5774f8d590e085275a0f541e6f03c943e9/examples/v1beta1/kubeflow-training-operator/xgboostjob-lightgbm.yaml#L46-L48

Reply:
For TFJob and PyTorchJob it should work for 1.1 and 1.2 release also, but we can keep support only for >= 1.3 version.
@kubeflow/wg-training-leads Do we want to keep support for previous versions for Training Operators in Katib ?

Reply:
Sorry, @andreyvelich.
I was able to verify that the above examples work with pytorch-operator. I withdraw the following comment.

https://github.com/kubeflow/katib/pull/1691#discussion_r721396767
> I think It needs to describe the training-operator version since I guess it only works with train-operator >= 1.3 in the following examples. WDYT @andreyvelich ?